olive/passes/onnx/qnn_preprocess.py

```diff
@@ -47,7 +99,18 @@ def _run_for_config(
         from onnxruntime.quantization.execution_providers.qnn import qnn_preprocess_model
 
         output_model_path = resolve_onnx_path(output_model_path, Path(model.model_path).name)
-        modified = qnn_preprocess_model(model.model_path, output_model_path, **config)
+
+        modified = qnn_preprocess_model(
```

Comment:
This also has a `external_data_location` https://github.com/microsoft/onnxruntime/blob/e93a860819545ea64acfe36e19e2b954389d48bf/onnxruntime/python/tools/quantization/execution_providers/qnn/preprocess.py#L24 which is equivalent to `location` in https://github.com/microsoft/Olive/blob/fed7c66ec1cb44105e57ca86242c35d79b4080ff/olive/passes/onnx/common.py#L118. 

could you add a similar logic to decide the value so that the `external_data_config` behaves the same for all passes?